src/newsroom-context/hooks/useGetTranslationUrl.ts

```diff
@@ -8,6 +8,32 @@ import { getCategoryHasTranslation, getCategoryUrl } from '../../utils';
 import { useCurrentCategory } from './useCurrentCategory';
 import { useCurrentStory } from './useCurrentStory';
 
+// Pulled from SDK types to not leak the `@prezly/sdk` dependency into the client bundle
+enum StoryVisibility {
+    PUBLIC = 'public',
+    EMBARGO = 'embargo',
+    PRIVATE = 'private',
+    CONFIDENTIAL = 'confidential',
+}
```

Comment:
Had to pull this enum in, so that `@prezly/sdk` won't leak into the client bundle.
@e1himself do you know any alternatives for such situations? I already did that in other places and it's not super optimal, but I can't seem to find a better solution.

Reply:
Currently Next.js seems to not do it. Maybe it's only enabled when the app itself is compiled as a module, but that currently doesn't work well on Next.js (although they claim to support it out of the box)

Reply:
> Currently Next.js seems to not do it. Maybe it's only enabled when the app itself is compiled as a module, but that currently doesn't work well on Next.js (although they claim to support it out of the box)

Ah, I've seen this before. I believe the modern bundlers only do tree-shaking for ESM code. But because our Javascript SDK is compiled to CommonJS only, it's not being performed out-of-the box.

I think that's a good reason to introduce a dual ESM/CommonJS build for the Javascript SDK. Or just go with ESM-only releases.

Reply:
Yeah, I've been thinking that the time has probably come to move all of our libraries to ESM (although it still feels shaky with some big libraries not really embracing it yet)